articles/vibe-coding-catfe-report.md

```
@@ -0,0 +1,142 @@
+
+Oikonです。外資系IT企業でソフトウェアエンジニアをしています。
+
+2025年7月29日-8月1日の4日間で、**RevenueCatがVibe Coding Catfe（バイブコーディングカフェ）というイベント**を開催しました！
+
+RevenueCatは、iOS・Androidアプリケーションのサブスクリプション管理と収益化を支援するプラットフォームを提供する米国企業です。ChatGPT、Notionなども採用しています。
+
+この記事では、イベントレポートと登壇した話についての備忘録を書くとともに、RevenueCatが開催する大規模ハッカソン**Shipaton**について、個人開発者向けに有益な情報を紹介します。
+
+## Shipaton
+
+https://revenuecat-shipaton-2025.devpost.com/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)